### PR TITLE
Feature: Add About Page

### PR DESCRIPTION
**Pull request by hampus-code**
In this PR:

1. Added about us page

### CODE REVIEW

**Review by carolCase — APPROVED** [review]
good job!

**Review by Clemond — APPROVED** [review]
Very good :D 👯

File: about.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+This is the about page for Restaurants
```
Comment (ahmadardal) [inline about.txt:1]: Can you please add a headline as well?

Comment (hampus-code) [reply about.txt:3]: Yes! I have added a headline

File: about.txt [hunk 2]
```diff
@@ -1 +1,3 @@
+Headline: Aboud Us
```
Comment (ahmadardal) [inline about.txt:1]: `Aboud`? 😂

Comment (ahmadardal) [reply about.txt]: Gj, approving

**Review by ahmadardal — APPROVED** [review]
👍

**Pull request merged**